packages/leva/src/components/Leva/Leva.tsx

```diff
@@ -40,7 +40,8 @@ export function useRenderRoot(isGlobalPanel: boolean) {
           document.getElementById('leva__root') || Object.assign(document.createElement('div'), { id: 'leva__root' })
         if (document.body) {
           document.body.appendChild(rootEl)
-          ReactDOM.render(<Leva isRoot />, rootEl)
+          const root = createRoot(rootEl)
+          root.render(<Leva isRoot />)
```

Comment:
I think I saw somewhere that using multiple roots is discouraged in React 18, would it be possible to switch to `createPortal()` instead? I guess the API might have to change, e.g. requiring consumers to render a `<Leva />` component

Reply:
I think this would actually be good API change where there's less "magic" and you can easily toggle it on and off.